scripts/check-coverage.jsh

```diff
@@ -43,7 +43,7 @@ var moduleCoverageData = List.of(
   new ModuleCoverage("sirius-components-diagrams-graphql", 29.0),
   new ModuleCoverage("sirius-components-gantt", 86.0),
   new ModuleCoverage("sirius-components-collaborative-gantt", 36.0),
-  new ModuleCoverage("sirius-components-gantt-graphql", 53.0),
+  new ModuleCoverage("sirius-components-gantt-graphql", 52.0),
```

Comment:
Why?

Reply:
I added some integration tests and updated the coverage consequently